base_tier_validation_forward/wizard/forward_wizard_view.xml

```diff
@@ -13,10 +13,7 @@
                         <field name="forward_reviewer_id" />
                         <field name="forward_description" required="1" />
                     </group>
-                    <group>
-                        <field name="has_comment" invisible="1" />
-                        <field name="approve_sequence" invisible="1" />
-                    </group>
+                    <group />
```

Comment:
Why do we need to keep this empty group?
If this is for reverse dependencies, we should probably add a comment on top explaining this (to be sure we don't delete it by mistake later on).